boards/wemos-zero/Makefile.include

```diff
@@ -0,0 +1,18 @@
+PORT_LINUX ?= /dev/ttyACM0
+PORT_DARWIN ?= $(firstword $(sort $(wildcard /dev/tty.usbmodem*)))
+
+CFLAGS += -DBOOTLOADER_UF2
+
+# setup serial terminal
+include $(RIOTMAKE)/tools/serial.inc.mk
+
+# setup the flash tool used
+ifeq ($(PROGRAMMER),jlink)
+  # in case J-Link is attached to SWD pins, use a plain CPU memory model
+  JLINK_DEVICE := atsamd21
+  include $(RIOTMAKE)/tools/jlink.inc.mk
```

Comment:
You could also move this to the common folder

Reply:
The problem is that the `arduino-zero` uses the `sam0.inc.mk`, which is not prepared for using a bootloader yet. I think I should just do so and could also update the `serpente` and other to use it.

Reply:
Can you verify that I didn't break anything for `serpente`? Do you by chance have an official Arduino Zero? It would be nice if someone could test if flashing still works there as before.

Reply:
`serpente` still flashes 🐍
I don't have any `arduino-zero` at my disposal though.